jobs/cloud_controller_ng/templates/ccng_monit_http_healthcheck.sh.erb

```diff
@@ -51,8 +51,8 @@ while true; do
   status=$?
   set -e
   if [[ $status > 0 ]] ; then
-    echo $(date --rfc-3339=ns) "ccng_monit_http_healthcheck failed to curl <${URL}>: exit code $status"
-    exit $status
+    echo "$(date --rfc-3339=ns) ccng_monit_http_healthcheck failed to curl <${URL}>: exit code $status"
+    [[ $status != 7 ]] && exit $status
```

Comment:
I'd prefer a actual proper if statement, but maybe that's bikeshedding.

Reply:
My preferred option is this syntax for one line, but it was changed to a classic if statement to improve readability.